src/services/user.service.js

```diff
@@ -0,0 +1,52 @@
+// const { v4 } = require('uuid');
+let users = [];
+let nextId = 1;
+
+const getAll = () => {
+  return users;
+};
+
+const getById = (id) => {
+  return users.find((user) => user.id === id) || null;
+};
+
+const updateUser = ({ id, name }) => {
```

Comment:
The updateUser function doesn't handle the case when the user is not found. It's better to add a check for this case and throw an error if the user doesn't exist.